docs/ORTModule_Training_Guidelines.md

```diff
@@ -124,6 +124,13 @@ Before full qualified name can be got from exporter, this environment variables
 	export ORTMODULE_SKIPPED_AUTOGRAD_FUNCTIONS="megatron.fp16.fp16.fused_kernels.GELUFunction"
 	```
 
+#### ORTMODULE_ENABLE_COMPUTE_OPTIMIZER
+
+- **Feature Area**: *ORTMODULE/Optimizations*
+- **Description**: By default, this is enabled then some computation can be saved. This env var can be used for disabling
+the optimization to guarantee exactly same compute with baseline (for example PyTorch, when doing convergence parity
+debugging).
+
```

Comment:
👍🏽 
Maybe in another PR, can we add some details about what compute optimizer will try to do with examples? Will be good for devs to understand the motivation and how to interpret the optimizer.

Reply:
+1

Reply:
Yes will add